api/azure/testdata/terraform-output.json

```diff
@@ -0,0 +1,200 @@
+{
+  "domain_name": {
+    "sensitive": false,
+    "type": "string",
+    "value": "ck8stest.azure-domain.com"
+  },
+  "sc_control_plane_lb_ip_address": {
+    "sensitive": false,
+    "type": "string",
+    "value": "89.145.167.47"
+  },
+  "sc_dns_name": {
+    "sensitive": false,
+    "type": [
+      "tuple",
+      [
+        "string",
+        "string",
+        "string",
+        "string",
+        "string",
+        "string"
+      ]
+    ],
+    "value": [
+      "*.ops.ck8stest.azure-domain.com",
+      "dex.ck8stest.azure-domain.com",
+      "grafana.ck8stest.azure-domain.com",
+      "harbor.ck8stest.azure-domain.com",
+      "kibana.ck8stest.azure-domain.com",
+      "notary.harbor.ck8stest.azure-domain.com"
+    ]
+  },
+  "sc_ingress_controller_lb_ip_address": {
+    "sensitive": false,
+    "type": "string",
+    "value": "89.145.167.76"
+  },
+  "sc_master_ips": {
+    "sensitive": false,
+    "type": [
+      "object",
+      {
+        "ck8stest-service-cluster-master-0": [
+          "object",
+          {
+            "private_ip": "string",
+            "public_ip": "string"
+          }
+        ]
+      }
+    ],
+    "value": {
+      "ck8stest-service-cluster-master-0": {
+        "private_ip": "10.0.10.205",
+        "public_ip": "159.100.242.187"
+      }
+    }
+  },
+  "sc_nfs_ips": {
```

Comment:
Do we have an nfs server on azure? I didn't find this anywhere else in the code :thinking: